app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -33,7 +33,19 @@
             <option value="blue">Blue</option>
         </select>
     </div>
-
+    <div class="user_categories">
```

Comment:
Is that for creating a new event or also edit event?

Is it possible to edit a category after creating it?